lightstep/recorder.py

```diff
@@ -223,15 +276,48 @@ def _add_span(self, span):
 
         Will delete a previously-added span if the limit has been reached.
         """
+        now_micros = util._now_micros()
+        
+        spanRecord = ttypes.SpanRecord(
+            trace_guid=str(span.context.trace_id),
+            span_guid=str(span.context.span_id),
+            runtime_guid=str(span._tracer.recorder._runtime_guid),
+            span_name=str(span.operation_name),
+            join_ids=[],
+            oldest_micros=long(span.start_time),
+            youngest_micros = long(now_micros),
+            attributes=[],
+            log_records=[]
+        )
+
+        for key in span.tags:
+            if key[:5] == "join:":
```

Comment:
nitpick: let's define `JOIN_ID_TAG_PREFIX = "join:"`, and use it here... both for the string, and also `len(JOIN_ID_TAG_PREFIX)` instead of `5`.